research/format.markdown

```diff
@@ -0,0 +1,124 @@
+| Deleted     | `phone`            | This SHOULD be a phone number (format MAY follow E.123)                                  |
+
+### Changes
+
+| Attribute   | Previous Description                                                                      | Present Description           |
+|-------------|-------------------------------------------------------------------------------------------|------------------------------------------------------------------------------------------------------------|
+| `date-time`   | This SHOULD be a date in ISO 8601 format of YYYY-MMDDThh:mm:ssZ in UTC time. This is... | A string instance is valid against this attribute if it is a valid date representation as defined by RFC 3339, section 5.6[RFC3339]                           |
+| `URI`         | SHOULD be a uri                                                                           | A string instance is valid against this attribute if it is a valid URI, according to [RFC3986]          |
+| `email`       | SHOULD be email                                                                           | A string instance is valid against this attribute if it is a valid Internet email address as defined by RFC 5322, section 3.4.1 [RFC5322]                        |
+| `ip-address`  | SHOULD be internet protocol version 4 address                                                                 | ipv4: A string instance is valid against this attribute if it is a valid representation of an IPv4 address according to the `dotted-quad` ABNF syntax as defined in RFC 2673, section 3.2 [RFC2673] |
+| `ipv6`        | SHOULD be internet protocol version 6 address                                                                 | A string instance is valid against this attribute if it is a valid representation of an IPv6 address as defined in RFC 2373, section 2.2 [RFC2373]                |
+| `host-name`   | This SHOULD be a host-name                                                               | A string instance is valid against this attribute if it is a valid representation for an Internet host name, as defined by RFC 1034, section 3.1 [RFC1034]           |
+
+## Draft 4 to Draft 6
+
+References: [Draft 4](https://json-schema.org/draft-04/draft-fge-json-schema-validation-00#rfc.section.7
+),
+[Draft 6](https://json-schema.org/draft-06/draft-wright-json-schema-validation-01#rfc.section.8)
+
+| Change Type | Format Attribute | Description                                                                                                          |
+|-------------|------------------|----------------------------------------------------------------------------------------------------------------------|
+| Added       | `uri-reference`    | This attribute applies to string instances. A string instance is valid against this attribute if it is a valid URI Reference (either a URI or a relative-reference), according to [RFC3986] |
```

Comment:
These RFC links seems to not be rendering

Reply:
Actually none of them are rendering

Reply:
I have changd it to render, I thought before it wasnt necessary